src/offlinelogstorage/dlt_offline_logstorage_behavior.c

```diff
@@ -383,10 +384,13 @@ int dlt_logstorage_storage_dir_info(DltLogStorageUserConfig *file_config,
             if (config->num_files == 1 && file_config->logfile_optional_counter) {
                 /* <filename>.dlt or <filename>_<tmsp>.dlt */
                 if ((files[i]->d_name[len] == suffix[0]) ||
+                    (files[i]->d_name[len] == gzsuffix[0]) ||
                     (file_config->logfile_timestamp &&
-                     (files[i]->d_name[len] == file_config->logfile_delimiter))) {
+                     (files[i]->d_name[len] ==
+                      file_config->logfile_delimiter))) {
                     current_idx = 1;
-                } else {
+                }
```

Comment:
Please do not touch the format but focusing on code change, it distracts me a lots during review.
Thanks.

Reply:
This was done automatically by the commit check